### PR TITLE
reduce repetitive spl_object_hash() at SendResponseEvent

### PR DESCRIPTION
**Pull request by samsonasik**
- [x] Is this related to quality assurance?

### CODE REVIEW

File: src/ResponseSender/SendResponseEvent.php [hunk 1]
```diff
@@ -66,9 +66,9 @@ public function setContentSent()
     {
         $response = $this->getResponse();
         $contentSent = $this->getParam('contentSent', []);
-        $contentSent[spl_object_hash($response)] = true;
+        $contentSent[$responseObjectHash = spl_object_hash($response)] = true;
```
Comment (Xerkus) [inline src/ResponseSender/SendResponseEvent.php]: Please extract variable assignment to a separate line

File: src/ResponseSender/SendResponseEvent.php [hunk 2]
```diff
@@ -93,9 +93,9 @@ public function setHeadersSent()
     {
         $response = $this->getResponse();
         $headersSent = $this->getParam('headersSent', []);
-        $headersSent[spl_object_hash($response)] = true;
+        $headersSent[$responseObjectHash = spl_object_hash($response)] = true;
```
Comment (Xerkus) [inline src/ResponseSender/SendResponseEvent.php]: Same, extract variable assignment to a separate line

**Review by Xerkus — CHANGES_REQUESTED** [review]

File: src/ResponseSender/SendResponseEvent.php (hunk 1 above)
Comment (samsonasik) [reply src/ResponseSender/SendResponseEvent.php]: @Xerkus done ;)

File: src/ResponseSender/SendResponseEvent.php (hunk 2 above)
Comment (samsonasik) [reply src/ResponseSender/SendResponseEvent.php]: @Xerkus done ;)

**Review by Xerkus — APPROVED** [review]

**Comment by Xerkus** [comment]
@samsonasik while we are at it, can you also add import for the spl function? Thanks

**Comment by samsonasik** [comment]
@Xerkus done ;)

**Comment by Xerkus** [comment]
@samsonasik thank you

**Pull request merged**